.github/workflows/upgrade-deps.yml

```diff
@@ -0,0 +1,59 @@
+name: Upgrade dependencies
```

Comment:
Again this is optional (and requires a separate `DEPLOY_KEY` ssh-key), but helps keep dependencies up-to-date in a way we've found more convenient than Dependabot.

Reply:
Fixed in https://github.com/oddbird/sass-site/pull/66